corpus/convert.py

```diff
@@ -177,6 +192,30 @@ def run(self):
                     else segment.recording.name
                 )
                 segment_track = segment.track + 1 if segment.track else 1
+
+                orth = f" {segment.orth.strip()} "
```

Comment:
I see you add the spaces here because the following `replace_recursive` needs `" {token} "` even at the start of sentence. But then you should remove these artificial whitespaces also if `self.fix_whitespace` is false.